packages/remote-replay-launcher/src/types.ts

```diff
@@ -16,6 +16,7 @@ export interface ExecuteRemoteTestRunOptions {
   onTunnelCreated?: (data: TunnelData) => void;
   onTestRunCreated?: (testRun: TestRun) => void;
   onProgressUpdate?: (testRun: TestRun) => void;
+  onTunnelLocked?: () => void;
```

Comment:
nit (can ignore for this): might be useful to also expose a `onTestRunCompleted` optional callback.

Reply:
Makes sense to add once we need it, but I'll leave this for now.